l10n_ar_afipws_fe/views/account_move_views.xml

```diff
@@ -7,31 +7,31 @@
          <field name="inherit_id" ref="l10n_ar.view_move_form"/>
         <field name="arch" type="xml">
             <!-- we change button labels for better usability -->
-            <button name="action_post" position="attributes">
+            <!-- <button name="action_post" position="attributes">
                 <attribute name="attrs">{'invisible': ['|', ('state', '!=', 'draft'), ('validation_type', '!=', False)]}</attribute>
                 <attribute name="states"></attribute>
-            </button>
+            </button> -->
             <button name="action_post" position="after">
-                <button name="action_post" type="object" attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('validation_type', '!=', 'production')]}" string="Validar en AFIP" class="oe_highlight" groups="account.group_account_invoice"/>
-                <button name="action_post" type="object" attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('validation_type', '!=', 'homologation')]}" string="Validar en HOMOLOGACION" class="oe_highlight" groups="account.group_account_invoice"/>
+                <button name="action_post" type="object" invisible="state '!= 'draft' or 'validation_type' != 'production'" string="Validar en AFIP" class="oe_highlight" groups="account.group_account_invoice"/>
+                <button name="action_post" type="object" invisible="state != 'draft' or 'validation_type' != 'homologation'" string="Validar en HOMOLOGACION" class="oe_highlight" groups="account.group_account_invoice"/>
             </button>
             <field name="l10n_ar_afip_concept" position="after">
                 <field name="afip_fce_es_anulacion"/>
-                <label for="afip_auth_code" string="AFIP associated period" attrs="{'invisible': [('move_type', 'not in', ['out_refund'])]}"/>
-                <div class="oe_inline"  attrs="{'invisible': [('move_type', 'not in', ['out_refund'])]}">
+                <label for="afip_auth_code" string="AFIP associated period" invisible="'move_type' not in 'out_refund'"/>
+                <div class="oe_inline"  invisible="'move_type' not in 'out_refund'">
                         <field name="afip_associated_period_from" class="oe_inline" placeholder="Date From"/> -
                         <field name="afip_associated_period_to" class="oe_inline" placeholder="Date to"/>
                 </div>
 
             </field>
             <notebook>
-                <page string="AFIP" name="afip" attrs="{'invisible': [('move_type', 'not in', ['out_invoice', 'out_refund'])]}">
+                <page string="AFIP" name="afip">
```

Comment:
creo que deberíamos conservar el invisible para los casos de facturas que no usan WS
`<page string="AFIP" name="afip" invisible="move_type not in ['out_invoice', 'out_refund']">`